microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -0,0 +1,106 @@
+---
+description: Use the Detached Elements tool in the Microsoft Edge DevTools to find and fix DOM memory leaks.
+title: Debug DOM memory leaks
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/29/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, detached elements, dom, memory, leak, node
+---
+
+# Debug DOM memory leaks with the Detached Elements tool
+
+A memory leak can occur in your application when an element is no longer attached to the DOM tree but is still referenced by some JavaScript running on the page. These elements are called **detached elements**. In order for the browser to garbage collect (GC) the detached element, there must be no references to the element from either the DOM tree or JavaScript code.
+
+The following sections show you how to use the **Detached Elements** tool in the Microsoft Edge DevTools to find detached elements that the browser cannot GC in a demo application and identify the JavaScript that is still referencing the detached element. By changing your JavaScript to release the element, you will be able to reduce the number of detached elements on your page.
+
+For more information about heap snapshots and detached elements, read [Discover detached DOM tree memory leaks with Heap Snapshots](/devtools-guide-chromium/memory-problems/#discover-detached-dom-tree-memory-leaks-with-heap-snapshots).
+
+## Open the Detached Elements tool
+
+The **Detached Elements** tool is available as an experiment since Microsoft Edge version 93 and is available by default in Microsoft Edge version 97. Check your version of Microsoft Edge by navigating to `edge://version`. If your version number is less than 97, read [Turning an experiment on or off
+](/devtools-guide-chromium/experimental-features/#turning-an-experiment-on-or-off) for instructions on turning the `Detached Elements` experiment on.
+
+To open the **Detached Elements** tool against our demo application, complete the following steps.
```

Comment:
```suggestion
To open the **Detached Elements** tool in the demo application, complete the following steps.
```

Reply:
To open the **Detached Elements** tool for the demo application: